docs/contracts/NFT Contract/NFT Actions/adjtrdwin.a.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'adjtrdwin.a'
+order: 5
+deploy: []
+---
+
+# adjtrdwin.a - adjust trading windows
+
+Allows a token manager to update the `trading_window_start` and `trading_window_end` for an existing token factory.
+
+## Technical Behavior
+
+Tradeability can only be update from this:
+
+-   Uniq is never Tradable (null, 0)
```

Comment:
Need to add clarification that `(null, 0)` is `trading_window_start` and `trading_window_end`.